src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
Both of them are very similar but not exactly the same. If the current working directory is `/relay/data` and the setting path is `/schema.json`:

`join(process.cwd(), config.file)` will return `/relay/data/schema.json`, on the other hand,  `resolve(config.file)` will return `/schema.json`, which is incorrect.